lib/facter/vmware_tools_uninstaller_path.rb

```diff
@@ -0,0 +1,3 @@
+Facter.add(:vmware_tools_uninstaller_path) do
```

Comment:
Please contain this fact.

And if you're going to execute shell commands, do so with `Facter::Core::Execution.execute()` as there is better error handling that way.

Could also consider using just Ruby for this instead of `which`: https://stackoverflow.com/a/5471032

Reply:
Or better yet, use [`Facter::Core::Execution.which()`](https://www.rubydoc.info/gems/facter/Facter%2FCore%2FExecution.which) as shown in the [example](https://puppet.com/docs/puppet/latest/fact_overview.html#example-slightly-more-complex-fact-confined-to-linux-with-a-block).